tests/conftest.py

```diff
@@ -457,7 +457,7 @@ def full_record(users):
                     "funder": {
                         "id": "00k4n6c32",
                     },
-                    "award": {"id": "00k4n6c32::755021"},
```

Comment:
we should change how the data goes in (i.e. the awards_v fixture) instead of the dump/expected data. To match https://github.com/inveniosoftware/invenio-vocabularies/blob/master/invenio_vocabularies/contrib/awards/datastreams.py#L53